### DISCUSSION TITLE
Python API查询报错

### DISCUSSION THREAD

**Discussion by OswinGuai** [apache/iotdb#4068] [Q&A]
用Python API查询数据，之前没有报错，今天却报了下面的异常：
<img width="1102" alt="截屏2021-10-02 上午9 55 49" src="https://user-images.githubusercontent.com/7878475/135700308-c5d851a3-5996-4c5e-a3ef-2ed7a76e730b.png">

同样的SQL语句用start-cli.sh查询就不会报错。

IoTDB的Python API版本试过0.12.1、0.12.2，都有这个错误。

**Comment by HTHou** [comment]
试试在sql语句里的时间上加上时区信息？
